Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
+
+gem 'xcpretty'
```

Comment:
This is ridiculously nitpicky, but on the ruby side, our style guide prefers double quotes as the default